extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/CommandSpec.scala

```diff
@@ -57,7 +57,8 @@ trait CommandSpecs[T <: CommandSpec] {
 case class DatabaseCommandSpec(
     classname: String,
     databaseDescs: Seq[DatabaseDesc],
-    opType: String = "QUERY") extends CommandSpec {}
+    opType: String = "QUERY",
+    uriDescs: Seq[UriDesc] = Nil) extends CommandSpec {}
```

Comment:
```suggestion
    uriDescs: Seq[UriDesc] = Nil,
    opType: String = "QUERY") extends CommandSpec {}
```
Shall we keep the order in `descs` for different kinds and followed by the `opType`?

Reply:
Hmmm if we want to do this, better in a separated pr and do this for all CommandSpec